File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV6000U00LaySummaryDetailInfo.java
package nta.med.data.model.ihis.invs;

public class INV6000U00LaySummaryDetailInfo {
	private String slipCode    ;
	private String jaeryoCode  ;
	private String jaeryoName  ;
	private String subul_danga  ;
	private String danui        ;
	private String jaegoQty    ;
	private String sougaku      ;
	public INV6000U00LaySummaryDetailInfo(String slipCode, String jaeryoCode, String jaeryoName, String subul_danga,
			String danui, String jaegoQty, String sougaku) {
		super();
		this.slipCode = slipCode;
		this.jaeryoCode = jaeryoCode;
		this.jaeryoName = jaeryoName;
		this.subul_danga = subul_danga;
		this.danui = danui;
		this.jaegoQty = jaegoQty;
		this.sougaku = sougaku;
	}
	public String getSlipCode() {
		return slipCode;
	}
	public void setSlipCode(String slipCode) {
		this.slipCode = slipCode;
	}
	public String getJaeryoCode() {
		return jaeryoCode;
	}
	public void setJaeryoCode(String jaeryoCode) {
		this.jaeryoCode = jaeryoCode;
	}
	public String getJaeryoName() {
		return jaeryoName;
	}
	public void setJaeryoName(String jaeryoName) {
		this.jaeryoName = jaeryoName;
	}
	public String getSubul_danga() {
		return subul_danga;
	}
	public void setSubul_danga(String subul_danga) {
		this.subul_danga = subul_danga;
	}
	public String getDanui() {
		return danui;
	}
	public void setDanui(String danui) {
		this.danui = danui;
	}
	public String getJaegoQty() {
		return jaegoQty;
	}
	public void setJaegoQty(String jaegoQty) {
		this.jaegoQty = jaegoQty;
	}
	public String getSougaku() {
		return sougaku;
	}
	public void setSougaku(String sougaku) {
		this.sougaku = sougaku;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV6000U00LaySummaryDetailInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Invs
{
    public class INV6000U00LaySummaryDetailInfo
    {
        private String _slipCode;
        private String _jaeryoCode;
        private String _jaeryoName;
        private String _subulDanga;
        private String _danui;
        private String _jaegoQty;
        private String _sougaku;

        public String SlipCode
        {
            get { return this._slipCode; }
            set { this._slipCode = value; }
        }

        public String JaeryoCode
        {
            get { return this._jaeryoCode; }
            set { this._jaeryoCode = value; }
        }

        public String JaeryoName
        {
            get { return this._jaeryoName; }
            set { this._jaeryoName = value; }
        }

        public String SubulDanga
        {
            get { return this._subulDanga; }
            set { this._subulDanga = value; }
        }

        public String Danui
        {
            get { return this._danui; }
            set { this._danui = value; }
        }

        public String JaegoQty
        {
            get { return this._jaegoQty; }
            set { this._jaegoQty = value; }
        }

        public String Sougaku
        {
            get { return this._sougaku; }
            set { this._sougaku = value; }
        }

        public INV6000U00LaySummaryDetailInfo() { }

        public INV6000U00LaySummaryDetailInfo(String slipCode, String jaeryoCode, String jaeryoName, String subulDanga, String danui, String jaegoQty, String sougaku)
        {
            this._slipCode = slipCode;
            this._jaeryoCode = jaeryoCode;
            this._jaeryoName = jaeryoName;
            this._subulDanga = subulDanga;
            this._danui = danui;
            this._jaegoQty = jaegoQty;
            this._sougaku = sougaku;
        }

    }
}